lib/rexml/parsers/baseparser.rb

```diff
@@ -157,6 +157,11 @@ module Private
           DEFAULT_ENTITIES_PATTERNS[term] = /&#{term};/
         end
         XML_PREFIXED_NAMESPACE = "http://www.w3.org/XML/1998/namespace"
+
+        QUOTE = [].tap { |x|
+          x['"'.ord] = '"'
+          x["'".ord] = "'"
+        }
```

Comment:
I'm OK with this concept but can we use `Hash` instead?

```suggestion
        QUOTE = {
          '"'.ord => '"',
          "'".ord => "'",
        }
```

Can we use better name for this? `QUOTES`? `QUOTE_CODE_POINT_TO_STRING`?

Reply:
Since we decided to use `case`, we will not use this code.
Thanks.